src/csv/area_and_circ_of_circles.py

```diff
@@ -0,0 +1,16 @@
+import csv
+import area_and_circum_calculations as acc
+
+
+with open('radius_of_circles.csv') as circles_file:
+    csv_reader = csv.reader(circles_file, delimiter=",")
+
+    current_position = 1
+
+    for line_array in csv_reader:
+        build_data = acc.do_build_data(line_array, current_position)
+
+        if build_data != "":
+            print(build_data)
+
+            current_position += 1
```

Comment:
The indentation here is wrong, current position should always loops.  However in you code the position only moves when the build_data is not blank.  Position should always increment by one on every iteration of the loop.  So the current_position += 1 in line 16 has to be outside the if block.

Reply:
fixed